MAINTAINERS.md

```diff
@@ -1,10 +1,10 @@
-# Maintainers
+# Project Maintainers
 
-This file lists the initial committers, and hence the maintainers of the Chaos Mesh project.
+This file lists the maintainers and committers of the Chaos Mesh project.
 
-In short, maintainers are people who are in charge of the maintenance of the Chaos Mesh project. This includes everything from day-to-day tasks such as issue triage to completing epics. Maintainers should be the first point of contact for security issues or other critical issues that require privacy not offered by the issue tracker.
+In short, maintainers are people who are in charge of the maintenance of the Chaos Mesh project.Committers are active community members who have shown that they are committed to the continuous development of the project through ongoing engagement with the community. For detailed description of the roles, see [Chaos Mesh Governance](https://github.com/chaos-mesh/chaos-mesh/blob/master/GOVERNANCE.md)
```

Comment:
...maintenance of the Chaos Mesh project. Committers       ...  

Add a space before `Committers`